src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -15,14 +15,14 @@ public class AdbRunner
     {
         #region Constructor and state variables
 
-
-
+        // When packaged as a DotNet CLI tool, this should always be a minimal adb.exe
+        private const string DefaultAdbExePath = @"..\..\..\content\adb.exe";

```

Comment:
I think you need `..\..\..\content\adb\adb.exe`

Reply:
yup

Reply:
Also would be great to use `Path.Join` so that we can support the unix world

Reply:
will add this when I add linux adb.

Reply:
(also for most things you can just use forward slashes in Windows and it figures it out :) )